bot/resources/tags/generators-iterators.md

```diff
@@ -0,0 +1,45 @@
+**Generators and Iterators**
+
+What is a generator and an iterator and what is the difference between each other?
+
+**Generator**
+
+A generator is a function that may contain a `yield` or sometimes `return` statements, a generator can also be an expression(genexpr).
```

Comment:
This seems problematic, generators don't have to be functions. You can create a generator class by implementing methods defined in `collections.abc.Generator`